resources/views/components/language-changer.blade.php

```diff
@@ -1,5 +1,5 @@
 @props([
-    'modelName' => 'name',
+    'modelName' => $model->getTable() === 'individuals' ? 'bio' : 'name',
```

Comment:
Maybe let's change this from `modelName` to something more appropriate — what we're doing is getting a test value to check that the model has been (somewhat) translated. So maybe even `testValue`?

Also I'd rather use `$model instanceOf App\Models\Individual` than `getTable()` as that's been our approach elsewhere.